apps/condo/domains/contact/hooks/useTableColumns.tsx

```diff
@@ -4,6 +4,10 @@ import { useIntl } from '@core/next/intl'
 import React, { useMemo } from 'react'
 import { createSorterMap, IFilters } from '../utils/helpers'
 import { getTextFilterDropdown, getFilterIcon } from '@condo/domains/common/components/TableFilter'
+import { Highliter } from '@condo/domains/common/components/Highliter'
+import { isEmpty } from 'lodash'
+import { Typography } from 'antd'
```

Comment:
it's better to put global import at the top

Reply:
done